assets/js/modules/analytics-4/utils/withCustomDimensions.test.js

```diff
@@ -89,6 +98,53 @@ describe( 'withCustomDimensions', () => {
 		expect( container ).toHaveTextContent( 'Insufficient permissions' );
 	} );
 
+	it( 'sets appropriate `redirectURL` in persmission error object if creating custom dimensions failed due to the user not having `EDIT_SCOPE`', () => {
```

Comment:
Typo (plus grammatical tweak):

```suggestion
	it( 'sets the appropriate `redirectURL` in the permission error object if creating custom dimensions failed due to the user not having `EDIT_SCOPE`', () => {
```